force-app/main/default/classes/SOQL.cls

```diff
@@ -163,9 +185,10 @@ public with sharing class SOQL {
      */
     public static Decimal getTotalRevenueByStage(String stage) {
         // Use AggregateQuery to get the total revenue
-
+        AggregateResult[] oppRevQuery = [SELECT SUM(Amount)amtSum FROM Opportunity WHERE StageName = :stage OR Amount > 10000 WITH SECURITY_ENFORCED];
+        Decimal oppRev = (Decimal) oppRevQuery[0].get('amtSum');
```

Comment:
Good name for your aggregate alias. I personally like to do operatorOfFieldname, or in this case sumOfAmount.